include/configurations/ConfigurationReader.h

```diff
@@ -187,13 +187,29 @@ class ConfigurationReader
         else
             LOG(INFO) << "DRGBTConfig::MAX_NUM_MODIFY_ATTEMPTS is not defined! Using default value of " << DRGBTConfig::MAX_NUM_MODIFY_ATTEMPTS;
 
-        if (DRGBTConfigRoot["STATIC_PLANNER_NAME"].IsDefined())
-            DRGBTConfig::STATIC_PLANNER_NAME = DRGBTConfigRoot["STATIC_PLANNER_NAME"].as<std::string>();
-        else
-            LOG(INFO) << "DRGBTConfig::STATIC_PLANNER_NAME is not defined! Using default value of " << DRGBTConfig::STATIC_PLANNER_NAME;
+        if (DRGBTConfigRoot["STATIC_PLANNER_TYPE"].IsDefined())
+        {
+            std::string static_planner_type = DRGBTConfigRoot["STATIC_PLANNER_TYPE"].as<std::string>();
+            if (static_planner_type == "RGBMT*")
```

Comment:
Unneeded newline

Reply:
Why not switch case? Of unordered_map of enum and string?

Reply:
OK, good idea, tnx!